synth.py

```diff
@@ -65,6 +66,12 @@
         'toISOString)')
 # [END fix-dead-link]
 
+# remove browser.js, it does not work with TypeScript yet
+os.unlink('dev/src/browser.js')
+os.unlink('dev/webpack.config.js')
+
 # Node.js specific cleanup
 subprocess.run(["npm", "install"])
 subprocess.run(["npm", "run", "fix"])
+os.chdir("dev")
+subprocess.run(["npx", "compileProtos", "src"])
```

Comment:
Should we also run https://github.com/googleapis/nodejs-firestore/blob/master/dev/protos/update.sh here? That generates our d.ts typings

Reply:
Yes I guess so (it's unrelated to this particular PR but it's a good place to call any custom scripts after generation).